### DISCUSSION TITLE
[Fixed] Re: 30.11.2019 Drumlist crash when synth missing

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1149] [Team Posts]
@spamatica Hey! I observed this, even though you applied a fix. Fixed now.
It crashed because the drum list was empty.
I rearranged a few things and added some new methods. Test OK now.
ChangeLog
`    - Referring to 30.11.2019: "Fixed crash in drumlist when synth plugin was missing."`
`      Observed crash still there immediately (the drum list was empty).`
`      Fixed and tested OK, saving and reloading, even if synth plugin is missing. The`
`       drum list is filled now. Even if the missing plugin was MESS fluidsynth (unlikely),`
`       the default map will be used since no note names are available from the synth.`

**Comment by spamatica** [comment]
Alright, cool!

Den tis 4 feb. 2020 kl 09:03 skrev Tim <notifications@github.com>:
